docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -2989,7 +3238,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.3"
+   "version": "3.10.12"
```

Comment:
Keep the changes to the notebook to the minimum. In this case, the Python version should not be changed.

Reply:
Done